src/python_testing/TC_CADMIN_1_24.py

```diff
@@ -0,0 +1,140 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
```

Comment:
There look basically the same with just differences in the success / failure times, right? Can you put these in the same file (you can have more than one test_ function in a file), and just parameterize the test function?

Reply:
Thank you, i did not realize I could do that.
I have removed the separate test modules and created one merged test module for both CADMIN_1_22 and CADMIN_1_24 tests.